google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/DefaultCloudSdkService.java

```diff
@@ -38,6 +41,9 @@ public String getCloudSdkHomePath() {
 
   @Override
   public void setCloudSdkHomePath(String cloudSdkHomePath) {
-    propertiesComponent.setValue(CLOUD_SDK_PROPERTY_KEY, cloudSdkHomePath);
+    if (!StringUtil.isEmpty(cloudSdkHomePath)
+        && CloudSdkUtil.containsCloudSdkExecutable(cloudSdkHomePath)) {
+      propertiesComponent.setValue(CLOUD_SDK_PROPERTY_KEY, cloudSdkHomePath);
```

Comment:
I know that there probably isn't a good reason to unset the cloud sdk to an invalid or empty path.  But I hesitate to not allow them to do it.